Changelog.md

```diff
@@ -13,13 +13,16 @@ Compiler Features:
 
 Bugfixes:
  * Inheritance: Fix incorrect error on calling unimplemented base functions.
- * isoltest: Added new keyword `wei` to express function value in semantic tests
  * Reference Resolver: Fix scoping issue following try/catch statements.
  * Standard-JSON-Interface: Fix a bug related to empty filenames and imports.
  * SMTChecker: Fix internal errors when analysing tuples.
  * Yul AST Import: correctly import blocks as statements, switch statements and string literals.
 
 
+Testing Infrastructure:
+ * isoltest: Added new keyword `wei` to express function value in semantic tests
```

Comment:
I would actually say we could remove this entirely - it's not exactly user-facing...